fatiando/gravmag/transform.py

```diff
@@ -262,6 +263,36 @@ def tga(x, y, data, shape, method='fd'):
     return res
 
 
+def tilt(x, y, data, shape):
+    """
+    Calculates the magnetic tilt, as defined by Miller and Singh (1994):
+    .. math::
+    tilt(f) = tan^{-1}(\\frac{\\frac{dT}{dz}}{\\sqrt{\\frac{dT}{dx}^2 +
+    \\frac{dT}{dy}^2}})
+    Parameters:
+    * x, y : 1D-arrays
+        The x and y coordinates of the grid points
+    * data : 1D-array
+        The potential field at the grid points
+    * shape : tuple = (ny, nx)
+        The shape of the grid
+    Returns:
+    * tilt : 1D-array
+        The tilt angle of the total field.
+    **References**
```

Comment:
I had to check  this because I never remember, the section headers (Parameters, Returns, References) are all normal case (no bold). 

The docstring conventions need some updating. I'm looking into using the numpy style docstrings because they look a bit better. It'll be some work to convert all of them but might be worth doing.